composer.json

```diff
@@ -13,8 +13,8 @@
         "php": ">=7.2.5",
         "ext-json": "*",
         "paragonie/sodium_compat": "^1.13",
-        "symfony/validator": "^5.1",
-        "guzzlehttp/guzzle": "^7.2",
+        "symfony/validator": "^4.4 || ^5.1",
+        "guzzlehttp/guzzle": "^6.5 || ^7.2",
```

Comment:
I believe Guzzle 5 still technically has security coverage even though it's not actively supported. But also OK with a followup to discuss that since it is EOLish. We can decide whether to invest the time or not later.

Reply:
Follow-up opened.